Date.ts

```diff
@@ -1,17 +1,27 @@
+import { isly } from "isly"
 import { DateSpan } from "./DateSpan"
 import { Locale } from "./Locale"
 
 export type Date = string
+//type Digit = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
+// Representing year better creates to many options
+// export type Date = `${number}-${
+// 	| Exclude<
+// 			`${`0${Exclude<Digit, "0">}` | `1${"0" | "1" | "2"}`}-${Exclude<`${"0" | "1" | "2"}${Digit}`, "00">}` | "30",
+// 			"02-30"
+// 	  >
+// 	| `${"01" | "03" | "05" | "07" | "08" | "10" | "12"}-31`}`
```

Comment:
Not sure what purpose this comments serves.

Reply:
I was trying out a more explicit typing of the strings but performance is not great and code complexity high.